### ISSUE TITLE
Mod refuses to load on the server, once again (1.12.2-5.0.3)

### ISSUE THREAD

**Issue by thegrumble** [Txuritan/devotion-irc#4]
It appears the problem outlined in #2 is back for 1.12.2 (5.0.3):

```
[10:40:30] [Server thread/DEBUG] [FML]: Examining file eirairc-mc1.12.2-5.0.3.jar for potential mods
[10:40:30] [Server thread/TRACE] [FML]: Located mcmod.info file in file eirairc-mc1.12.2-5.0.3.jar
[10:40:30] [Server thread/DEBUG] [FML]: Identified a mod of type Lnet/minecraftforge/fml/common/Mod; (net.blay09.mods.eirairc.EiraIRC) - loading
[...]
[...]
[10:41:38] [IRC (chat.local)/INFO] [net.blay09.mods.eirairc.irc.IRCConnectionImpl]: > NICK eira
[10:41:38] [IRC (chat.local)/INFO] [net.blay09.mods.eirairc.irc.IRCConnectionImpl]: > USER EiraIRC "" "" :EiraIRC Bot
[10:41:38] [IRC (chat.local)/INFO] [net.blay09.mods.eirairc.irc.IRCConnectionImpl]: < :chat.local NOTICE * :*** Looking up your hostname...
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at net.blay09.mods.eirairc.util.Utils.addMessageToChat(Utils.java:59)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at net.blay09.mods.eirairc.EiraIRC$1.addChatMessage(EiraIRC.java:89)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at net.blay09.mods.eirairc.EiraIRC$1.addChatMessage(EiraIRC.java:81)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at net.blay09.mods.eirairc.handler.IRCEventHandler.firePrivateChatEvent(IRCEventHandler.java:201)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at net.blay09.mods.eirairc.irc.IRCConnectionImpl.handleMessage(IRCConnectionImpl.java:505)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at net.blay09.mods.eirairc.irc.IRCConnectionImpl.run(IRCConnectionImpl.java:236)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]:     at java.lang.Thread.run(Thread.java:748)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1061]: Caused by: java.lang.ClassNotFoundException: net.minecraft.client.Minecraft
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1061]:     at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1061]:     at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1061]:     at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[10:41:38] [IRC (chat.local)/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1061]:     ... 7 more
```

**Comment by Txuritan** [comment]
This should be an easy fix, to be truthful I think I updated the wrong version (one before the fix), I should have a fix up in a few hours.

**Comment by Txuritan** [comment]
It has been fixed with version 5.0.4 and is now on CurseForge for download.
